src/utils/vega2dot.js

```diff
@@ -0,0 +1,1058 @@
+import {EventStream, Operator, Dataflow} from 'vega-dataflow';
```

Comment:
Let's write this in ts.

Reply:
Sounds good. I just copied over what was previously in the `vega-inspector` for now.